app/src/main/java/co/nimblehq/compose/crypto/ui/common/AppDialogPopUp.kt

```diff
@@ -7,18 +7,22 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
 import co.nimblehq.compose.crypto.ui.theme.*
 
 @Composable
 fun AppDialogPopUp(
     onDismiss: () -> Unit,
-    onClick: () -> Unit,
     title: String,
     message: String,
-    actionText: String,
+    actions: List<DialogActionModel>,
```

Comment:
In our case, I think we only need a dialog with 2 buttons 🤔 So it could be `positiveButton, negativeButton`, passing the list of model quite confused.

Reply:
IMO, sometimes we might need only `Ok` button, sometimes we might want one `negative` and one `positive` buttons, and sometimes we might need other buttons. By passing actions, we can customize the action buttons to whatever we want. What do you think?